src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -99,7 +101,7 @@ export class Conversations extends React.Component<Props, State> {
         })
 
         if (!this.props.me.conversations.pageInfo.hasNextPage) {
-          this.setState({ completed: true })
+          this.setState({ completed: true, initialLoadDone: true })
```

Comment:
We really need these test accounts with data at various states to test our code more thoroughly 😩... soon